sources/packages/backend/libs/utilities/src/date-utils.ts

```diff
@@ -246,3 +246,48 @@ export function hasSomePeriodOverlap(periods: Period[]): boolean {
   }
   return false;
 }
+
+/**
+ * Validates if the date provided is within given days
+ * before now.
+ * e.g. If the current date is 10-JAN-2023 and
+ * given number of days is 4, it validates if the given date
+ * is on or after 06-JAN-2023.
+ * @param givenDate date to be validated.
+ * @param numberOfDays number of days before today.
+ * @returns result if the given date is after given days before now.
+ */
+export const isAfterGivenDaysBeforeNow = (
```

Comment:
I noticed that this file has a mixed of exporting const arrow methods and sometimes exporting the methods.
Can we try to stick with one of them only?
I would suggest exporting the method.